CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this project will be documented in this file - [read more
 
 ## [Unreleased]
 
+### Added
+- Tracer limited mode, stopping span creation when memory use raises to 80% of current PHP memory limit #437
```

Comment:
Can we update this to mention the new function names and new env var to configure it? :)

Reply:
I was thinking we shouldn't advertise this at this moment too much. But if you feel its important to do so then I'll add these.

Reply:
I see your point on this. I think the env var should definitely be documented but putting the functions here I'll leave up to you. For me, I always like to have a heads up if an extension is going to put more unnamespaced function definitions in the root symbol table - just to be aware of it. But like you said, we might not want to put that out there quite yet until we nail down a hard API for everything. :)

Reply:
oh but all our functions are namespaced ;)  Using good old C type namespacing of prefixing everything with `dd_trace`. So clashes with existing functions should be minimal. Do you think us adding function to main namespace might have other sideffects ?

Reply:
Hehe. Yes, we are doing some good old-fashioned PHP-style namespacing. 😄I just like to be explicit in the documentation - especially when adding new symbols to the symbol table. That's all. 😺I'm OK leaving this out until we nail down a solid API.